nsv13.dme

```diff
@@ -1568,7 +1568,7 @@
 #include "code\modules\cargo\bounties\item.dm"
 #include "code\modules\cargo\bounties\mech.dm"
 #include "code\modules\cargo\bounties\medical.dm"
-#include "code\modules\cargo\bounties\mining.dm"
+//#include "code\modules\cargo\bounties\mining.dm" //NSV13
```

Comment:
This doesn't work. The compiler will automatically clear this line. Simply leave the file unticked.

Reply:
so just leave #include "code\modules\cargo\bounties\mining.dm" be?